README.md

```diff
@@ -1,24 +1,24 @@
-# Single cell explorer
+# CZ CELLxGENE Explorer
 _an interactive explorer for single-cell transcriptomics data_
 
-This repo is for the CZI-hosted version, aka single-cell-explorer, or simply `explorer`. The public 'desktop' version [is here](https://github.com/chanzuckerberg/cellxgene).
+This repo contains a fork of CZ CELLxGENE Annotate called CZ CELLxGENE Explorer which is deployed as part of CZ CELLxGENE Discover to enable exploration of datasets published on that platform.
 
 [![Push Tests](https://github.com/chanzuckerberg/single-cell-explorer/workflows/Push%20Tests/badge.svg)](https://github.com/chanzuckerberg/single-cell-explorer/actions?query=workflow%3A%22Push+Tests%22)
 ![Code Coverage](https://codecov.io/gh/chanzuckerberg/single-cell-explorer/branch/main/graph/badge.svg)
 
-cellxgene (pronounced "cell-by-gene") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the [Human Cell Atlas](https://humancellatlas.org). Leveraging modern web development techniques to enable fast visualizations of at least 1 million cells, we hope to enable biologists and computational researchers to explore their data.
+CZ CELLxGENE (pronounced "cell-by-gene") is an interactive data explorer for single-cell transcriptomics datasets, such as those coming from the [Human Cell Atlas](https://humancellatlas.org). Leveraging modern web development techniques to enable fast visualizations of at least 1 million cells, we hope to enable biologists and computational researchers to explore their data.
 
-Whether you need to visualize one thousand cells or one million, cellxgene helps you gain insight into your single-cell data.
+Whether you need to visualize one thousand cells or one million, CZ CELLxGENE helps you gain insight into your single-cell data.
```

Comment:
```suggestion
Whether you need to visualize one thousand cells or one million, CELLxGENE Explorer helps you gain insight into your single-cell data.
```

Per comments elsewhere, we're trying to establish CELLxGENE as the brand family, so are eliminating uses of that name to refer to individual applications. Here the correct referent is "Explorer"